src/averageAges.js

```diff
@@ -55,7 +74,24 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const motherSon = people.map(function(child) {
```

Comment:
```suggestion
  const motherSon = people.map((child) => {
```
It is better to use arrow functions in callbacks